examples/misc/test/language_tour/classes_test.dart

```diff
@@ -61,7 +66,7 @@ void main() {
   test('const, identical, runtimeType', () {
     _test() {
       // #docregion const
-      var p = const ImmutablePoint(2, 2);
+      final p = const ImmutablePoint(2, 2);
```

Comment:
Why this change from `var` to `final`?

Reply:
I was wondering if you'd catch that. :) I don't remember why I changed it. I think it was a side effect of the DartPad changes I was making. I'll change it back.

Reply:
:)
👍